other/bootstrap_daemon/tox-bootstrapd.service

```diff
@@ -4,11 +4,8 @@ After=network.target
 
 [Service]
 Type=forking
-PermissionsStartOnly=true
-ExecStartPre=-/bin/mkdir /var/run/tox-bootstrapd -p
-ExecStartPre=/bin/chown tox-bootstrapd:tox-bootstrapd -R /var/run/tox-bootstrapd
 WorkingDirectory=/var/lib/tox-bootstrapd
-ExecStart=/usr/local/bin/tox-bootstrapd /etc/tox-bootstrapd.conf
+ExecStart=/usr/bin/tox-bootstrapd /etc/tox-bootstrapd.conf
```

Comment:
Don't replace `local` with `bin`. We assume that you build the code yourself and install it yourself, and self-built programs go into `local` by default.